frontend/src/global_styles/content/work_packages/tabs/_activities.sass

```diff
@@ -50,7 +50,7 @@
 
 .comments-number
   position: absolute
-  right: 0
+  right: 5px
```

Comment:
This affects the desktop screen sizes as well, resulting in the numbers being not aligned with the sub-headers any more:
<img width="65" alt="Bildschirmfoto 2022-04-19 um 13 07 07" src="https://user-images.githubusercontent.com/7457313/163990690-33cf5600-9135-44c7-80ac-5f6479c93152.png">